### DISCUSSION TITLE
Submitting Weekly Assignments

### DISCUSSION THREAD

**Discussion by bst3ck** [USU-ITLS-x265-23-summer/class-repo#41] [Q&A]
I was not sure if we still needed to submit the assignment in Canvas or if we just need to do the assignment in Github. It appears some of the grading was happening as I was completing some of the tasks, but I was not sure if we needed to officially submit the assignment in canvas.

Thanks

**Comment by metasean** [comment]
There aren't any assignments that require you to use Canvas.  

I've got a list of methods to demonstrate different criteria here: https://github.com/orgs/USU-ITLS-x265-23-summer/discussions/16

I'll make a post once I've completed grading for a given week.  If you think I've overlooked something, drop me a Canvas email message with the rubric and a link (or other reference) to where it's been completed.

That said, I will be completing the rubric scoring in Canvas.  As you're sure to have noticed now, it's not real-time.  You may see progress on several criteria but not others; it's just because of how I'm manually batching the checks.
